torchtune/training/checkpointing/_checkpointer.py

```diff
@@ -467,8 +467,26 @@ def load_checkpoint(self) -> Dict[str, Any]:
                 tie_word_embeddings=self._config["tie_word_embeddings"],
             )
         elif self._model_type == ModelType.LLAMA3_VISION:
-            raise NotImplementedError(
-                "Safe tensors support for LLaMA3 Vision will be added soon. Pleaee use the FullModelMetaCheckpointer for now."
+            from torchtune.models.llama3_2_vision._convert_weights import (
+                llama3_vision_hf_to_tune,
```

Comment:
What's the reason for gating the import here? Due to transitive torchvision dep or something?